README.md

```diff
@@ -31,9 +31,9 @@ To add a new brand:
 
 # Publishing
```

Comment:
We use "releasing" everywhere else. Is there a slight connotation difference or they're perfect synonyms?